simulator/src/main.rs

```diff
@@ -1,10 +1,21 @@
+mod defined;
```

Comment:
This is not a descriptive name, something like `simulation_config` would be much better.